### PR TITLE
Create static elements within dashboard, logs page and some basic functionality

### LINKED ISSUE #163: Dashboard : Create remaining static elements, including logs page
### Describe the feature

Currently the dashboard does not contain any functionality in relation to allowing exploration and selection of specific groups and/or filter based on monitor status.

This would be a useful functionality that would not only help debug things even further but also offers a more personalized way of seeing the logs and uptime.

Logs are also constantly being spit by our backend to provide an easy to debug what is currently happening. For that end, we need to create a way of visualizing and analysing these logs in the frontend, hence why the creation of this page is a necessity.

### Use Case

- Whenever I see something odd within the logs graph or feel like exploring the logs, I want to be capable of analysing them in the dashboard to understand what's going on.
- If something goes awry, I want a place to track the logs so that I can quickly fix it.
- I want to be capable of filtering the dashboard by status, so that I can debug things easier.

### Proposed Solution

Creating the UI in a static manner is a good step to understanding if this would work as expected.
The UI design has already been created and is now only waiting for the implementation within frontend.

### Other Information

N/A

### PR DESCRIPTION
**Pull request by pdPinho**
### Issue

Closes #163 

### Reason for this change

Our dashboard was lacking quite a lot of both visual elements and functionnality. This PR aims to solve that.

### Description of changes

- Added react query;
- Added static elements to dashboard;
- Added logic to some dashboard elements;
- Added static logs page;
- Fixed some small bugs within pagetitle.

### Description of how you validated changes

Ran containers without a problem.

### Checklist
- [X] I have tested my changes locally.
- [ ] I have updated the documentation to reflect my changes.
- [ ] I have added/updated unit tests (if applicable).
- [X] My code adheres to the [CONTRIBUTING GUIDE](https://ua-smart-signage-platform.github.io/Documentation-Website/guidelines/)

### Aditional Information
N/A

### CODE REVIEW

File: frontend/src/routes/Dashboard/index.jsx [hunk 1]
```diff
@@ -1,14 +1,266 @@
...
+            },
+        },
+        cells: {
+            style: {
+                paddingLeft: '10px',
+                paddingRight: '0px',
+                wordBreak: 'break-word',
+            }
+        }
+    };
```
Comment (fungame2270) [inline frontend/src/routes/Dashboard/index.jsx]: meter estas configs num ficheiro a parte dentro do mesmo folder que sao usadas

File: frontend/src/routes/Dashboard/index.jsx [hunk 2]
```diff
@@ -1,14 +1,266 @@
...
+        {
+            name: (                
+                <div className="flex flex-row">
+                     Online for
+                </div>
+            ),
+            selector: row => row.online,
+            width: "160px"
+        }   
+    ];
```
Comment (fungame2270) [inline frontend/src/routes/Dashboard/index.jsx]: e estas

**Review by fungame2270 — CHANGES_REQUESTED** [review]

**Comment by pdPinho** [comment]
Test coverage and sonarcloud reliability will be ignored for this PR due to the upcoming ones being mostly focused on refactoring, deep analysis and/or overhaul.

**Review by fungame2270 — APPROVED** [review]
lgtm

**Pull request merged**